pages/about.js

```diff
@@ -17,7 +18,57 @@ import {
 import { ContentSection } from "@components/content-section";
 import { PageHeader } from "@components/page-header";
 import { Type } from "@components/typography";
-const contributorsData = require("nteract-members");
+
+type Member = {
+  name: ?string,
+  login: string,
+  websiteUrl: ?string,
+  avatarUrl: string,
+  url: string
+};
+
+async function getMembers(organisation: string) {
```

Comment:
We probably want this moved into a folder just for this data fetching since we only need it run server side and cached most of the time (can just send up the json).